CONTRIBUTING.md

```diff
@@ -17,13 +17,14 @@
     1. [Updating consul-helm](#updating-consul-helm)
     1. [Testing a new CRD](#testing-a-new-crd)
     1. [Update Consul K8s acceptance tests](#update-consul-k8s-acceptance-tests)
-1. [Adding a new ACL Token](#adding-a-new-acl-token)
-1. [Testing the Helm chart](#testing-the-helm-chart)
+3. [Adding a new ACL Token](#adding-a-new-acl-token)
```

Comment:
we should keep these as is. markdown does the right things with the indexing if you keep them all at "1"